packages/graphql-migrations/src/abstract/generateAbstractDatabase.ts

```diff
@@ -232,7 +234,7 @@ class AbstractDatabaseBuilder {
 
       // Object
     } else if (isObjectType(fieldType)) {
-      columnName = annotations.name || `${field.name}Id`
+      columnName = relationshipMarker?.key || annotations.name || `${field.name}Id`
```

Comment:
That could be nightmare to document. Do we need both annotations and marker support at the time?

Reply:
We discussed supporting the existing annotation but just documenting the new one in Graphback.